README.md

```diff
@@ -209,11 +238,19 @@ stores = [
   }
 ]
 
-most_expensive_products_by_store = []
+
+
 
 ```
 
-### Answer
+### Answer still not working 
+most_expensive_product_in_stores = stores.map do |store|
+    most_expensive_product = store [:products].max_by do |product|
+    p product [:price]
```

Comment:
this will print out all the products' prices
print the `most_expensive_product_in_stores` instead